api-tests/features/ticf/failed-ticf-requests.feature

```
@@ -0,0 +1,45 @@
+@Build
+Feature: Failed TICF requests
+  Scenario Outline: TICF Management API request configured for 400 initially and no CI in reuse journey
+    Given there is an existing TICF record for the user with details
+      | responseDelay | 0                            |
+      | type          | RiskAssessment               |
+      | txn           | randomUuid                   |
+      | statusCode    | <statusCode>                          |
```

Do we need all of these? It might be more obvious what you're interested in if you omit all the defaults
- `responseDelay` of zero is the default
- are `txn` and `type` ever anything else?

I'm torn as to whether it'd be cleaner just do drop the datatable and have some simpler steps (`Given TICF CRI will respond with a {number} statusCode` etc.) but it makes it more flexible this way.

---

An undefined txn means the request has timed out but I have updated it to default to a random string unless it's specified in the table to be an empty string (which is the only case where we'd specify the txn) 

Also decided to keep the table for flexibility but it does mean that we'd still need to define an empty table like this: :')
```
Given TICF CRI will respond with default parameters
      | | |
```

---

I don't think you need it at all if it's all defaults - the TICF CRI stub should respond with a 'default' success response if you don't prime it with anything :)

---

(unless you specifically want to remove an existing setup)

---

ohh I see, thanks :)